client/src/components/formHelpers/Form.test.js

```diff
@@ -8,7 +8,10 @@ import Form from './Form';
 
 const defaultTestProps = {
   buttonText: 'Submit',
-  formFields: ['name', 'website'],
+  formFields: [
+    { field: 'name', fieldLabel: 'name label' },
```

Comment:
I recommend having a `fieldLabel` with some words capitalized and some not, so we make we properly display the project titles in the settings page.  This would mean that you will need to update the corresponding regular expression to not use the `i` flag.